QSB/TransformSync/SectorSync.cs

```diff
@@ -11,6 +11,23 @@ public class SectorSync : MonoBehaviour
         private Dictionary<uint, Transform> _playerSectors;
         private Sector[] _allSectors;
         private MessageHandler<SectorMessage> _sectorHandler;
+        private readonly List<Sector.Name> SectorWhitelist = new List<Sector.Name>(new Sector.Name[]{
```

Comment:
Since you (likely) won't add or remove sectors to the whitelist, you can use the array directly:
`private readonly Sector.Name[] SectorWhitelist = { ... }`

Reply:
@amazingalek It was just so I could use List.Contains for the check. What's a good way to do that with a normal array?

Reply:
Arrays have .Contains :-)
Just add `using System.Linq;`